docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -233,17 +173,183 @@ specific alias. For example, at most 1 tier can have the alias HDD. If you want
+data when Alluxio storage is full. This is termed eviction.
+
+There are two modes of eviction in Alluxio, asynchronous (default) and synchronous. You can switch
+between the two by enabling and disabling the space reserver which handles asynchronous eviction.
+For example, to turn off asynchronous eviction:
+
+```
+alluxio.worker.tieredstore.reserver.enabled=false
+```
+
+Asynchronous eviction is the default implementation of eviction. It relies on a periodic space reserver 
+thread in each worker to evict data. It waits until the worker storage utilization reaches a configurable
+ high watermark. Then it evicts data based on the eviction policy until it reaches the configurable low
+  watermark. For example, if we had the same 16+100+100=216GB storage configured, we can set eviction to
+  kick in at around 200GB and stop at around 160GB:
 
-These are the configuration parameters for tiered storage.
+```
+alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9 # 216GB * 0.9 ~ 200GB
+alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75 # 216GB * 0.75 ~ 160GB
+```
+
+In write or read-cache heavy workloads, asynchronous eviction can improve performance.
+
+Synchronous eviction waits for a client to request more space than is currently available on the worker 
+and then kicks off the eviction process to free up enough space to serve that request. This leads to many 
+small eviction attempts, which is less efficient but maximizes the utilization of available Alluxio space.
+
+### Evictors
+
+Alluxio uses evictors for deciding which blocks to evict, when space needs to be
+freed. Users can specify the Alluxio evictor to achieve fine grained control over the eviction
+process.
+
+Alluxio supports custom evictors. Out-of-the-box implementations include:
+
+* **GreedyEvictor**
+
+    Evicts arbitrary blocks until the required size is freed.
+
+* **LRUEvictor**
+
+    Evicts the least-recently-used blocks until the required size is freed.
+
+* **LRFUEvictor**
+
+    Evicts blocks based on least-recently-used and least-frequently-used with a configurable weight.
+    If the weight is completely biased toward least-recently-used, the behavior will be the same as
+    the LRUEvictor.
+
+* **PartialLRUEvictor**
+
+    Evicts based on least-recently-used but will choose StorageDir with maximum free space and
+    only evict from that StorageDir.
+
+In the future, additional evictors will be available. Since Alluxio supports custom evictors,
+you can also develop your own evictor appropriate for your workload.
+
+When using synchronous eviction, it is recommended to use smaller block sizes (around 64-128MB),
+to reduce the latency of block eviction. When using the space reserver, block size does not affect
+eviction latency.
+
+## Operate Data in Alluxio
+
+TODO(Zac): Add documentation on check capacity, fraction of used capacity and etc
+TODO(Zac): Add documentation for free, load, and persist commands.
```

Comment:
TTL seems like it should be in this list?

Also, how are these sections different from the sections in the CLI documentation?

If someone asks a question "how do I use the [TTL|free|persist] feature", which page do we refer them to?

Reply:
In here I am thinking a high-level overview, maybe through a running example to show how to manage data in Alluxio with the  simplest use of these CLIs. The CLI doc is more comprehensive on the detailed syntax.

Reply:
@apc999 
That means this page should include all the ways you can "manage data" in Alluxio, and explain what it means and the implications, right? For example, to free a file, this page should discuss the "free" concept, and then present how to free a file. The CLI command for free is only 1 way to do that, and there is a java api for it as well.

Shouldn't this model be followed for all possible "data management" operations like TTL, free, persist, etc?

Reply:
@gpang  as we discussed, we will move the Java-level examples.